catalog/RUB/RUB_RUBCLIM_LCZ_global_lcz_map_v1.jsonnet

```diff
@@ -0,0 +1,162 @@
+local id = 'RUB/RUBCLIM/LCZ/global_lcz_map/v1';
+local subdir = 'RUB';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Global map of Local Climate Zones',
+  version: 'v1',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    Since their introduction in 2012, Local Climate Zones (LCZs) emerged as a new standard
+    for characterizing urban landscapes, providing a holistic classification approach that
+    takes into account micro-scale land-cover and associated physical properties.
+
+    This global map of Local Climate Zones, at 100m resolution and representative for the
+    nominal year 2018, is derived from multiple earth observation datasets and expert
+    LCZ class labels. The LCZ scheme complements other land use / land cover schemes by
+    its focus on urban and rural landscape types, which can be described by any of the 17
+    classes in the LCZ scheme. Out of the 17 LCZ classes, 10 reflect the 'built'
+    environment, and each LCZ type is associated with generic numerical descriptions
+    of key urban canopy parameters critical to model atmospheric responses to urbanisation.
+    In addition, since LCZs were originally designed as a new framework for urban
+    heat island studies, they also contain a limited set (7) of 'natural' land-cover
+    classes that can be used as 'control' or 'natural reference' areas.
+
+    As these seven natural classes in the LCZ scheme can not capture the heterogeneity of
+    the world’s existing natural ecosystems, we advise users - if required - to combine
+    the built LCZ classes with any other land-cover product that provides a wider range
+    of natural land-cover classes.
+
+    See also:
+
+    * [LCZ Typology](https://doi.org/10.1175/BAMS-D-11-00019.1)
+
+    * [Global map of LCZs](https://doi.org/10.5194/essd-14-3835-2022)
+
+    * [Global map of LCZs - dataset](https://doi.org/10.5281/zenodo.6364594)
+
+    * [LCZ Gaussian filtering](https://doi.org/10.1038/s41597-020-00605-z)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    {
+      rel: ee_const.rel.source,
+      href: 'https://doi.org/10.5194/essd-14-3835-2022',
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.5281/zenodo.6364594',
+    },
+  ],
+  keywords: [
+    'Local Climate Zones',
```

Comment:
please use lowercase tags with no spaces (use _ instead) and sort the tags

Reply:
Done.